metadata/src/main/java/org/apache/kafka/metadata/authorizer/StandardAuthorizerData.java

```diff
@@ -104,6 +116,10 @@ private static Logger createLogger(int nodeId) {
         return new LogContext("[StandardAuthorizer " + nodeId + "] ").logger(StandardAuthorizerData.class);
     }
 
+    private static Logger auditLogger() {
```

Comment:
Is there a reason not to just make this static, like

`static final Logger auditLog = ...` ?

We used to make all the loggers static prior to the desire to use LogContexts (which this doesn't)...